src/test/test_bitcoin.h

```diff
@@ -50,7 +50,7 @@ struct BasicTestingSetup {
  */
 class CConnman;
 struct TestingSetup: public BasicTestingSetup {
-    CCoinsViewDB *pcoinsdbview;
+    std::unique_ptr<CCoinsViewDB> pcoinsdbview;
```

Comment:
In commit "Use unique_ptr for pcoinscatcher/pcoinsdbview/pcoinsTip/pblocktree"

Can this variable just be deleted like the one in rpcnestedtests.h, since it is shadowing a global with the same name? If it can't be deleted, it would be good to have a comment noting that it does shadow the global (and maybe explaining why).